sdk/tables/azure-data-tables/azure/data/tables/_shared_access_signature.py

```diff
@@ -89,9 +94,12 @@ def generate_account(
             or address range specified on the SAS token, the request is not authenticated.
             For example, specifying sip=168.1.5.65 or sip=168.1.5.60-168.1.5.70 on the SAS
             restricts the request to those IP addresses.
-        :param Union[str, SASProtocol] protocol:
+        :param protocol:
             Specifies the protocol permitted for a request made. The default value
             is https,http. See :class:`~azure.cosmosdb.table.common.models.Protocol` for possible values.
```

Comment:
```suggestion
            is https,http. See :class:`~azure.data.tables.SASProtocol` for possible values.
```
That default value is also incorrect.